From 73a8524d1456f70d5743370af94282660eb5481d Mon Sep 17 00:00:00 2001
From: Lukas Czerner <lczerner@redhat.com>
Date: Thu, 23 Oct 2014 09:53:48 -0400
Subject: [fs] ext4: drop set but otherwise unused variable from ext4_add_dirent_to_inline()

Message-id: <1414058125-4183-75-git-send-email-lczerner@redhat.com>
Patchwork-id: 98178
O-Subject: [RHEL 7.1 PATCH v2 074/171] ext4: drop set but otherwise unused variable from ext4_add_dirent_to_inline()
Bugzilla: 1150139
RH-Acked-by: Eric Sandeen <sandeen@redhat.com>

From: Azat Khuzhin <a3at.mail@gmail.com>

BZ 1150139
https://bugzilla.redhat.com/show_bug.cgi?id=1150139

BREW 8142710
https://brewweb.devel.redhat.com/taskinfo?taskID=8142710

Upstream commit 5ba052fe33807893a26076d2ac95cf31d5e0b578

 Signed-off-by: Azat Khuzhin <a3at.mail@gmail.com>
 Signed-off-by: "Theodore Ts'o" <tytso@mit.edu>
 Signed-off-by: Lukas Czerner <lczerner@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/ext4/inline.c b/fs/ext4/inline.c
index e78e362..0559ed4 100644
--- a/fs/ext4/inline.c
+++ b/fs/ext4/inline.c
@@ -994,11 +994,9 @@ static int ext4_add_dirent_to_inline(handle_t *handle,
  struct inode *dir = dentry->d_parent->d_inode;
  const char *name = dentry->d_name.name;
  int  namelen = dentry->d_name.len;
- unsigned short reclen;
  int  err;
  struct ext4_dir_entry_2 *de;
 
- reclen = EXT4_DIR_REC_LEN(namelen);
  err = ext4_find_dest_de(dir, inode, iloc->bh,
     inline_start, inline_size,
     name, namelen, &de);
-- 
1.7.1